source/Jobbr.Server.UnitTests/ShouldDateTimeExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using Shouldly;
+
+namespace Jobbr.Server.UnitTests
+{
+    public static class ShouldDateTimeExtensions
+    {
+        private static readonly TimeSpan DefaultOffsetTolerance = TimeSpan.FromSeconds(1);
+
+        public static DateTime ShouldBeUtcNow(this DateTime actual, string customMessage = null)
+        {
+            return actual.ShouldBeUtcNow(DefaultOffsetTolerance, customMessage);
```

Comment:
If we use a default tolerance we should probably rename ShouldBeUtcWithTolerance otherwise it's not clear that it's an approximated comparison